granary/microformats2.py

```diff
@@ -293,6 +295,17 @@ def json_to_object(mf2):
     'tags': [json_to_object(cat) for cat in props.get('category', [])],
   }
 
+  lat, lng = prop.get('latitude'), prop.get('longitude')
+  if lat and lng:
+    try:
+      lat, lng = float(lat), float(lng)
+      # 2 leading places for latitude, 3 for longitude
+      obj['position'] = '%0+10.6f%0+11.6f/' % (lat, lng)
```

Comment:
you can actually leave this blank; it already gets populated in `Source.postprocess_object()`. your formatting is great though, feel free to add it there if you want!